rubocop-airbnb/config/rubocop-gemspec.yml

```diff
@@ -7,3 +7,6 @@ Gemspec/RequiredRubyVersion:
   Enabled: false
   Include:
     - '**/*.gemspec'
+
+Gemspec/RubyVersionGlobalsUsage:
```

Comment:
This is enabled by default -- is it worth explicitly enabling here or should we just use the rubocop default settings?

Reply:
Let's explicitly enable it here for clarity. I don't know where the Rubocop defaults are set but I see tons of enabled cops in our configs that are probably also enabled by default.